src/main/java/org/apache/commons/lang3/CharEncoding.java

```diff
@@ -23,14 +23,14 @@
 /**
  * <p>Character encoding names required of every implementation of the Java platform.</p>
  *
- * <p>According to <a href="http://docs.oracle.com/javase/1.3/docs/api/java/lang/package-summary.html#charenc">JRE character
+ * <p>According to <a href="https://docs.oracle.com/javase/6/docs/api/java/lang/package-summary.html#charenc">JRE character
```

Comment:
There are no character encoding name at that URL; did you mean to point to https://docs.oracle.com/javase/8/docs/technotes/guides/intl/encoding.doc.html ?

Reply:
I found the original page on https://web.archive.org/web/20111127040714/http://docs.oracle.com/javase/1.3/docs/api/java/lang/package-summary.html.

I found that https://docs.oracle.com/javase/6/docs/api/java/nio/charset/Charset.html now contains an equivalent documentation (paragraph _Standard charsets_). I will update the link accordingly.